### PR TITLE
Added sample code for locally

### PR DESCRIPTION
**Pull request by yadavan88**

### CODE REVIEW

**Comment by yadavan88** [comment]
retest this please

File: scala3-lang-2/src/test/scala/com/baeldung/scala3/locally/LocallyBlockTest.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  it should "accidentally make a code block without newline as part of class body" in {
+    class MyClass
+
+    lazy val myClass = new MyClass {
+      println("This should be an independant block")
+    }
+
+    // it should compile since print method is accidentally considered as part of MyClass
```
Comment (dominiqueplante) [inline scala3-lang-2/src/test/scala/com/baeldung/scala3/locally/LocallyBlockTest.scala:24]: Should this be uncommented?

Comment (yadavan88) [reply scala3-lang-2/src/test/scala/com/baeldung/scala3/locally/LocallyBlockTest.scala]: This test itself should not be there, as we are only pritnting the content. Nothing to verify. This is added in the main class already

Comment (yadavan88) [reply scala3-lang-2/src/test/scala/com/baeldung/scala3/locally/LocallyBlockTest.scala]: Thanks for catching this :)

**Pull request merged**